src/plots/stackedBarPlot.ts

```diff
@@ -54,6 +60,92 @@ namespace Plottable.Plots {
       return this;
     }
 
+    protected _setup() {
+      super._setup();
+      this._labelArea = this._renderArea.append("g").classed(StackedBar._STACKED_BAR_LABEL_AREA_CLASS, true);
+      this._measurer = new SVGTypewriter.Measurers.CacheCharacterMeasurer(this._labelArea);
+      this._writer = new SVGTypewriter.Writers.Writer(this._measurer);
+    }
+
+    protected _drawLabels() {
+      super._drawLabels();
+
+      // remove all current labels before redrawing
+      this._labelArea.selectAll("g").remove();
+
+      const baselineValue = +this.baselineValue();
```

Comment:
the baseline value isn't _technically_ numeric, but I'm operating under the assumption that you need a numeric secondary scale in order to have a stacked bar that works correctly